### PR TITLE
Feat/goto page

### LINKED ISSUE #738: [Ember-Migration] Create Debug page
### Requirements
As we are migrating towards Emberjs we need the given section to be sync with our production, to look it exactly like production version. In this ticket we have to make the mentioned section UI as well as functionalities to match same as in production version.

#### In vanilla site (currently running www site in production, created with plain HTML, CSS, JS)
Debug page -  https://www.realdevsquad.com/debug

#### In beta site(created with EmberJS)
Doesn't exists

### Important links

- https://staging-www.realdevsquad.com/ (This one is made with Ember js, staging one)
- https://www.realdevsquad.com/ (This one is the production site)

### Additional Note

- Design should be same as in prod.
- Take reference from the prod site wherever needed.
- It should be responsive.
- Code quality should be maintained.
- Use `ember generate` commands to generate `components`, `templates` and other files.

### Questions you can have in mind

1. Is this should be under feature flag?
Yes, it should be under feature dev=true

2. What is feature flag?
It's a simple variable which we can use to turn off/on the feature. Currently we use dev=true in URL query.

3. Do you have to use the same code as in production site?
No, the solution is upto you how you want to approach the problem. It should be same UI wise and functionality wise, rest other implementation is on you.

4. In which branch ember code is in?
It's in `develop-ember` branch, make sure to checkout to that branch.

### PR DESCRIPTION
**Pull request by satyam73**
### What is the change?

Added goto page for redirection.

##### Example :

- Done this
- Made these changes

### Is Development Tested?

- [x] Yes
- [ ] No

### Before / After Change Screenshots

image
>Desktop

image
> Tablet

image
> Mobile
**Note :**
What to delete and what not to

- Keep the headings and answer the questions.
- Delete the examples
- Delete the description below headings `What is the change?` & `Before/After Screenshots`

### CODE REVIEW

**Comment by shubhamsigdar1** [comment]
> ### Issue
> ##### Example : Closes #<issue_number>
> ### What is the change?
> Provide a small description in points of what did you change
> 
> ##### Example :
> * Done this
> * Made these changes
> 
> ### Is Development Tested?
> * [x]  Yes
> * [ ]  No
> 
> ### Before / After Change Screenshots
> For visual or interaction changes. Please provide a video / screenshot.
> 
> ##### Example:
> Before : Image/Video
> 
> After : Image/Video
> 
> **Note :** What to delete and what not to
> 
> * Keep the headings and answer the questions.
> * Delete the examples
> * Delete the description below headings `What is the change?` & `Before/After Screenshots`

Fill description

File: app/constants/urls.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const APP_URLS = {
     PROFILE: `${SCHEME}staging-my.${DOMAIN}`,
     TASKS: `${SCHEME}staging-my.${DOMAIN}/tasks`,
     IDENTITY: `${SCHEME}staging-my.${DOMAIN}/identity`,
-    API_BACKEND: `${SCHEME}staging-api.${DOMAIN}`,
+    API_BACKEND: `http://localhost:3000`,
```
Comment (shubhamsigdar1) [inline app/constants/urls.js:38]: revert changes

File: app/templates/goto.hbs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{{page-title 'Please wait'}}
+
+<section class='redirection-wrapper'>
+  <img
+    class='logo-img'
+    src='/assets/images/rds-logo-2x.png'
+    alt='Real Dev Squad'
+  />
+  <h1 class='redirection'>Redirecting to your correct location...</h1>
+</section>
```
Comment (shubhamsigdar1) [inline app/templates/goto.hbs:10]: Write test

File: app/constants/urls.js (hunk 1 above)
Comment (satyam73) [reply app/constants/urls.js:38]: This is development config that's why added this

File: app/templates/goto.hbs (hunk 2 above)
Comment (satyam73) [reply app/templates/goto.hbs:10]: Will add msw for testing this in other PR

**Review by ajoykumardas12 — APPROVED** [review]

**Review by shubhamsigdar1 — DISMISSED** [review]

**Review by shubhamsigdar1 — COMMENTED** [review]
merge conflict

**Review by ajoykumardas12 — APPROVED** [review]

**Pull request merged**